### PR TITLE
Add unity lumin support for editor 2019.2.8f1

### PR DESCRIPTION
**Pull request by JipSterk**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not [already refused].
- [x] Checked the cask is submitted to [the correct repo].
- [x] Attempted to find an [appcast].

[token reference]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/Homebrew/homebrew-cask/pulls
[already refused]: https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues
[the correct repo]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256
[appcast]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md

### CODE REVIEW

File: Casks/unity-lumin-support-for-editor.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+cask 'unity-lumin-support-for-editor' do
+  version '2019.2.8f1,ff5b465c8d13'
+  sha256 '565ac77d4b1398d770276130a32338807be95678bb51e86fcf46c088672e0e06'
+
+  url "https://download.unity3d.com/download_unity/#{version.after_comma}/MacEditorTargetInstaller/UnitySetup-Lumin-Support-for-Editor-#{version.before_comma}.pkg"
+  appcast 'https://public-cdn.cloud.unity3d.com/hub/prod/releases-darwin.json'
+  name 'Unity Lumin Build Support'
+  homepage 'https://unity3d.com/unity/'
```
Comment (reitermarkus) [inline Casks/unity-lumin-support-for-editor.rb]: Is there a better homepage? Where did you find the download link?

Comment (JipSterk) [reply Casks/unity-lumin-support-for-editor.rb]: all current `unity-*-support-for-editor` use the same configuration.

Comment (reitermarkus) [reply Casks/unity-lumin-support-for-editor.rb]: Then those should also be changed to a better URL. It's not obvious to me how to find the download link from https://unity3d.com/unity/.

Comment (JipSterk) [reply Casks/unity-lumin-support-for-editor.rb]: unity decided to use modules to make the install size smaller.
the editor you can get from multiple pages [download](https://store.unity.com/download). [other download](https://unity3d.com/unity/whats-new/2019.2.8) here you also find all the separate modules for the editor. so what would you like?

Comment (reitermarkus) [reply Casks/unity-lumin-support-for-editor.rb]: `https://unity3d.com/unity/whats-new/` seems like a good choice.

**Comment by JipSterk** [comment]
@reitermarkus i have updated all the homepage urls but ci blocks changes to multiple cask files

**Comment by reitermarkus** [comment]
Yes, please put the other homepage changes in another PR.

**Comment by reitermarkus** [comment]
Thanks!

**Pull request merged**